src/design-system/components/chip/Chip.tsx

```diff
@@ -0,0 +1,233 @@
+import React, { CSSProperties, useState } from 'react'
+import classNames from 'classnames'
```

Comment:
Use `cn` function instead. You are importing package that is not present in package.json, (it used by another package, so technically it is present in node_modules). But if you need some package - always install it via `npm install...`. But for this example - use our `cn` function.

Reply:
didn't notice that, thanks